tests/app/v2/notifications/test_push_broadcast_notifications.py

```diff
@@ -164,77 +127,9 @@ def test_returns_201_after_read_timeout(
         response = post_send_push_broadcast_notification(client, sample_api_key(service), PUSH_BROADCAST_REQUEST)
         assert response.status_code == 201
 
-    @pytest.mark.parametrize(
-        'payload, personalisation, app',
-        [
-            (PUSH_BROADCAST_REQUEST, None, DEAFULT_MOBILE_APP_TYPE.value),
-            (
-                {**PUSH_BROADCAST_REQUEST, 'personalisation': {'foo': 'bar'}, 'mobile_app': MobileAppType.VETEXT.value},
-                {'foo': 'bar'},
-                MobileAppType.VETEXT.value,
-            ),
-        ],
-    )
-    def test_makes_call_to_vetext_client(
-        self,
-        client,
-        sample_api_key,
-        sample_service,
-        vetext_client,
-        payload,
-        personalisation,
-        app,
-    ):
-        """
-        App SIDs should be loaded from the environment.  For testing, the is .local.env.
-        """
-
+    @pytest.mark.parametrize('side_effect, expected_status', [(CeleryError, 502), (OperationalError, 502)])
+    def test_celery_returns_502(self, client, sample_api_key, sample_service, mocker, side_effect, expected_status):
```

Comment:
You don't need "expected_status" because it's always 502 according to the actual values and the test name.

Reply:
Updated.